src/imgtools/dicom/crawl/parse_dicoms.py

```
@@ -466,3 +470,57 @@
             )
 
     return barebones_dict
+
+def remove_duplicate_entries(
+        slim_db: list[dict[str, str]],
+        ignore_keys: list[str] = ["SubSeries"]
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)